pkg/types/ovirt/defaults/machinepools.go

```diff
@@ -0,0 +1,42 @@
+package defaults
+
+import (
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/ovirt"
+)
+
+const (
+	//DAffinyGroupBase is a holder which will be replaced with the cluster name when creating the affinity group
+	DAffinyGroupBase                 = "CLUSTERNAME"
```

Comment:
I strongly discourage using this as a means of coming up with an affinity group that includes the cluster name. Instead, I recommend adding the cluster name to the beginning or end of the name provided by the user. At a minimum, if you go the route of using a string to be replaced, then you need to document that around the type and add validation that enforces the existing of the string exactly once in the name.

Reply:
@Gal-Zaidman any thoughts on this?

Reply:
Well currently we use the replace to add the cluster name at the beginning.
The reason I used the place holder is that if a user inputs a custom name then it will not be replaced and if the default configuration is applied then the string will be replaced.
Instead, I can add some if to check if the affinity group name is different then the default affinity group name...

Reply:
Oh, so you are *not* expecting the user to include "CLUSTERNAME" somewhere in the affinity group name. That was not clear to me. How is the destroyer going to clean up affinity groups with user-supplied names then?

Reply:
I think you are right on second thought, there is no harm in always adding the cluster name to a new affinity group that the installer creates, it will simplify the logic and improve the destroy logic :)
Change that part, thanks for the feedback!